tools/clang/test/HLSLFileCheck/passes/hl/sroa_hlsl/NodeInput_type.ll

```diff
@@ -34,87 +50,76 @@ define void @loadStress_16(%"struct.DispatchNodeInputRecord<loadStressRecord>"*
 entry:
   %val.i = alloca i32, align 4
   %0 = call %dx.types.NodeHandle @"dx.hl.createnodeoutputhandle..%dx.types.NodeHandle (i32, i32)"(i32 11, i32 0)
-  %1 = call %dx.types.NodeHandle @"dx.hl.annotatenodehandle..%dx.types.NodeHandle (i32, %dx.types.NodeHandle, %dx.types.NodeInfo)"(i32 16, %dx.types.NodeHandle %0, %dx.types.NodeInfo { i32 6, i32 116 })
+  %1 = call %dx.types.NodeHandle @"dx.hl.annotatenodehandle..%dx.types.NodeHandle (i32, %dx.types.NodeHandle, %dx.types.NodeInfo)"(i32 16, %dx.types.NodeHandle %0, %dx.types.NodeInfo { i32 6, i32 128 })
   %2 = call %"struct.NodeOutput<loadStressRecord>" @"dx.hl.cast..%\22struct.NodeOutput<loadStressRecord>\22 (i32, %dx.types.NodeHandle)"(i32 9, %dx.types.NodeHandle %1)
   store %"struct.NodeOutput<loadStressRecord>" %2, %"struct.NodeOutput<loadStressRecord>"* %loadStressChild
   %3 = call %dx.types.NodeRecordHandle @"dx.hl.createnodeinputrecordhandle..%dx.types.NodeRecordHandle (i32, i32)"(i32 13, i32 0)
-  %4 = call %dx.types.NodeRecordHandle @"dx.hl.annotatenoderecordhandle..%dx.types.NodeRecordHandle (i32, %dx.types.NodeRecordHandle, %dx.types.NodeRecordInfo)"(i32 17, %dx.types.NodeRecordHandle %3, %dx.types.NodeRecordInfo { i32 97, i32 116 })
+  %4 = call %dx.types.NodeRecordHandle @"dx.hl.annotatenoderecordhandle..%dx.types.NodeRecordHandle (i32, %dx.types.NodeRecordHandle, %dx.types.NodeRecordInfo)"(i32 17, %dx.types.NodeRecordHandle %3, %dx.types.NodeRecordInfo { i32 97, i32 128 })
   %5 = call %"struct.DispatchNodeInputRecord<loadStressRecord>" @"dx.hl.cast..%\22struct.DispatchNodeInputRecord<loadStressRecord>\22 (i32, %dx.types.NodeRecordHandle)"(i32 11, %dx.types.NodeRecordHandle %4)
   store %"struct.DispatchNodeInputRecord<loadStressRecord>" %5, %"struct.DispatchNodeInputRecord<loadStressRecord>"* %inputData
-  %6 = alloca %"struct.GroupNodeOutputRecords<loadStressRecord>"
```

Comment:
Where is the HLSL for this? My expectation is that somehow this change made it so that it realized it didn't need two allocas for the same struct off the bat. This led to the elimination of the memcpy's which caught my attention. I worried that they might be related to what this test is trying to capture.

Reply:
The hlsl file is mentioned in a comment at the bottom.  I think it would be easier to notice if that comment was moved near the top of the file, instead.

See: https://github.com/microsoft/DirectXShaderCompiler/pull/5893/files#diff-e1f9916f53b273f0d0e8ded4023b2b024a5c7811294e8cd181313c6dcb36792bR197
```ll
; python3 ExtractIRForPassTest.py -p scalarrepl-param-hlsl -o outtest_without_nodeiotypecheck.ll %HLSL_SRC_DIR%\tools\clang\test\HLSLFileCheck\hlsl\workgraph\called_function_arg_record_object.hlsl -- -T lib_6_8
```